sycl/source/stream.cpp

```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include <climits>
```

Comment:
@jchlanda, please, move this include after current includes and put empty line before `#include <climits>` to suppress clang-format re-ordering.
Including standard headers after SYCL headers allows us to detect cases when standard header includes are missing in SYCL headers.

Reply:
Done, thank you for explaining the reasoning.